Makefile

```diff
@@ -62,28 +69,50 @@ dep:
 	export DEBIAN_FRONTEND=noninteractive && \
 	export TZ=America/New_York && \
 	apt-get update && \
-	apt-get -y install libgtest-dev && \
-	apt-get install -y gstreamer1.0-tools && \
-	apt-get install -y libgstreamer1.0-dev \
-		libgstreamer-plugins-base1.0-dev \
-		libgstreamer-plugins-good1.0-dev \
-		libgstreamer-plugins-bad1.0-dev
+	if [ "$(TARGET)" = "jetson" ]; then \
+		apt-get -y install libgtest-dev && \
+		apt-get install -y gstreamer1.0-tools && \
+		apt-get install -y libgstreamer1.0-dev \
+			libgstreamer-plugins-base1.0-dev \
+			libgstreamer-plugins-good1.0-dev \
+			libgstreamer-plugins-bad1.0-dev; \
+	elif [ "$(TARGET)" = "pi" ]; then \
+		apt-get install -y --no-install-recommends software-properties-common && \
+		apt-add-repository 'deb http://archive.raspberrypi.org/debian/ bullseye main' && \
+		wget -qO - https://archive.raspberrypi.org/debian/raspberrypi.gpg.key | apt-key add - && \
+		apt-get -y update && \
+		apt-get -y install libcamera0 libgstreamer1.0-dev libgstreamer1.0-0 gstreamer1.0-x gstreamer1.0-tools gstreamer1.0-plugins-base gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly libgstreamer-plugins-base1.0-dev && \
+		apt-get -y install libgtest-dev && \
+		cd /usr/src/gtest && \
+		cmake CMakeLists.txt && \
+		make && \
+		cp lib/*.a /usr/lib && \
```

Comment:
I installed Gtest yesterday on my amd64, and this was _not_ needed. It would just error because the apt command  `mv`'ed all of the archive files to `/usr/lib` for me, so there was nothing to do.

Reply:
Good call out here. There is no `make install` target, however, the make command is already copying to `/lib` so no need to move again to `/usr/lib`. Simplified the logic here a bit.

A simple apt install command does not do the job on bullseye/arm64 - does not actually fill libraries, rather, installs source code to `/usr/src`. 

Looking at the recommendations from the GoogleTest docs [here](https://google.github.io/googletest/quickstart-cmake.html) - they recommend using `FetchContent` to pull dependencies from github. I could work on refactoring to this pattern, however, I would prefer to stick with fetching the libraries from local lib path.

Reply:
This, minus the copying, looks fine to me. This is how I had to install it locally to get things working, too.